### PR TITLE
chrome: backfill RTCIceServer.urls from .url

### PR DESCRIPTION
**Pull request by fippo**
apparently there is a significant number of folks who still have
not upgraded from RTCIceServer.url to RTCIceServer.urls.
Let adapter do this for them.

cc @foolip

### CODE REVIEW

**Comment by fippo** [comment]
see also https://bugs.chromium.org/p/chromium/issues/detail?id=659133

**Comment by jan-ivar** [comment]
With the exception of https://github.com/webrtc/adapter/pull/411 our motto for adapter.js has been to polyfill the spec on older browsers, not the other way around.

This would be the second issue to deviate from that, which may be fine, but we should have a plan to tear both band-aids off in a major version bump a year from now, so this library doesn't become an extended-support library, contributing to the spread of non-spec API usage.

File: src/js/chrome/chrome_shim.js [hunk 1]
```diff
@@ -165,6 +165,28 @@ var chromeShim = {
...
+    } else {
+      // migrate from non-spec RTCIceServer.url to RTCIceServer.urls
+      var OrigPeerConnection = RTCPeerConnection;
+      window.RTCPeerConnection = function(pcConfig, pcConstraints) {
+        if (pcConfig && pcConfig.iceServers) {
+          for (var i = 0; i < pcConfig.iceServers.length; i++) {
+            var server = pcConfig.iceServers[i];
+            if (server.hasOwnProperty('url') &&
+                !server.hasOwnProperty('urls')) {
+              server.urls = server.url;
```
Comment (jan-ivar) [inline src/js/chrome/chrome_shim.js]: Don't land this without at least a deprecation warning to console!

Also note you're modifying the passed-in object here. Might wanna modify a copy.

**Comment by fippo** [comment]
@jan-ivar this is mostly an attempt to drive down the numbers for non-spec usage that @foolip is seeing (unsure if we can impact them) in order to justify removal of the non-spec behaviour.

Whether people update adapter often enough for this to make an impact... :-/

File: src/js/chrome/chrome_shim.js (hunk 1 above)
Comment (fippo) [reply src/js/chrome/chrome_shim.js]: done. Also added a console.warn inside the Edge shim. I assume we don't need it in Firefox

**Comment by KaptenJansson** [comment]
How do we know the devs who are using non spec apis are using adapter? :)

**Comment by fippo** [comment]
we don't. But as @jan-ivar said in the bug we can probably attribute the 60% urls to showing the right stuff in the samples!

**Comment by KaptenJansson** [comment]
I see. LGTM

**Pull request merged**